ChordProgression.h

```diff
@@ -0,0 +1,30 @@
+/******************************************************************
+    ChordProgression.h
+
+    structure to encapsulate a chord progression across the guitar fretboard
+******************************************************************/
+#pragma once
+#include <iostream>
```

Comment:
you aren't using anything from iostream in this header file.
don't include what you don't use.
- [ ] revise

Reply:
copy paste error fixed.